@navikt/core/react/src/util/i18n/locales/nb.ts

```diff
@@ -44,4 +44,9 @@ export default {
       labelSuffix: "slett",
     },
   },
+  ProgressBar: {
+    progress: "{value} av {valueMax}",
+    progressUnknown:
+      "Framdrift kan ikke beregnes, antatt tid er {seconds} sekunder.",
```

Comment:
Disse er vel begge like godkjent, men føler "fremdrift" er mer rett (??)
```suggestion
      "Fremdrift kan ikke beregnes, antatt tid er {seconds} sekunder.",
```

Reply:
I think this is just a matter of what you are used to. For example, Morten likes the "fram" variant iirc. It doesn't matter much to me, as long as we are consistent.